src/roll20/renderer.js

```diff
@@ -1,47 +1,47 @@
-
-class Roll20Displayer {
-    async sendMessage(request, title, html, buttons, character, whisper, play_sound, source, attributes, description, attack_rolls, roll_info, damage_rolls, total_damages, open) {
-        const req = {
-            action: "rendered-roll",
-            request,
-            title,
-            html,
-            buttons,
-            character,
-            whisper,
-            play_sound,
-            source,
-            attributes,
-            description,
-            attack_rolls,
-            roll_info,
-            damage_rolls,
-            total_damages,
-            open
-        }
-        handleRenderedRoll(req);
-    }
-    displayError(message) {
-        alertify.error(message);
-    }
-}
-
-class Roll20Roller {
-    roll(formula, data) {
-        return new DNDBRoll(formula, data);
-    }
-    async resolveRolls(name, rolls) {
-        return Promise.all(rolls.map(roll => roll.roll()))
-    }
-}
-
-class Roll20Prompter {
-    async prompt(title, html, ok_label = "OK", cancel_label = "Cancel") {
-        return new Promise((resolve, reject) => {
-            alertify.Beyond20Prompt(title, html, ok_label, cancel_label, resolve);
-        });
-    }
-}
-
-var roll_renderer = new Beyond20RollRenderer(new Roll20Roller(), new Roll20Prompter(), new Roll20Displayer());
+
+class Roll20Displayer {
+    async sendMessage(request, title, html, buttons, character, whisper, play_sound, source, attributes, description, attack_rolls, roll_info, damage_rolls, total_damages, open) {
+        const req = {
+            action: "rendered-roll",
+            request,
+            title,
+            html,
+            buttons,
+            character,
+            whisper,
+            play_sound,
+            source,
+            attributes,
+            description,
+            attack_rolls,
+            roll_info,
+            damage_rolls,
+            total_damages,
+            open
+        }
+        handleRenderedRoll(req);
+    }
+    displayError(message) {
+        alertify.error(message);
+    }
+}
+
+class Roll20Roller {
+    roll(formula, data) {
+        return new DNDBRoll(formula, data);
+    }
+    async resolveRolls(name, rolls) {
+        return Promise.all(rolls.map(roll => roll.roll()))
+    }
+}
+
+class Roll20Prompter {
+    async prompt(title, html, ok_label = "OK", cancel_label = "Cancel") {
+        return new Promise((resolve, reject) => {
+            alertify.Beyond20Prompt(title, html, ok_label, cancel_label, resolve);
+        });
+    }
+}
+
+var roll_renderer = new Beyond20RollRenderer(new Roll20Roller(), new Roll20Prompter(), new Roll20Displayer());
```

Comment:
any changes here? or it's all whitespace/CRLF changes ?

Reply:
No content changes were made to oll20/renderer.js. Just CRLF > LF.